src/internals/utils.ts

```diff
@@ -113,6 +113,10 @@ export function getOwnParamInfo(fn: any): ParamInfo[] {
  */
 export function getParamInfo(fn: any): ParamInfo[] | null {
   if (!Reflect.getMetadata(paramInfoKey, fn)) {
+    const orgComponentSym = (window as any).tabris.symbols.originalComponent;
```

Comment:
cannot run tests locally because it complains about `error no-undef : 'window' is not defined.`. How can I run tests locally?